samples/mongo/src/main/resources/static/resources/js/jquery/jquery.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v1.8.2 jquery.com | jquery.org/license */

```

Comment:
Thank you for pointing this out. I noticed that and included that detail in my summary https://github.com/spring-projects/spring-session/pull/371#issuecomment-187409315 (along with an issue to get the boot sample fixed).